setup-scripts/bot_utils.py

```diff
@@ -10,7 +10,7 @@
 sc_precision = 10 ** 24
 
 CHANNEL_NAME = "skynet-portal-health-check"
-ROLE_NAME = "skynet-prod"
+ROLE_NAME = "everyone"
```

Comment:
`ROLE_NAME` should be another environment variable so it can be set however you'd like. We use `skynet-prod` so only a subset of people on our discord get notified instead of everyone